Make & Name 4 ships

```diff
@@ -0,0 +1,67 @@
+#include <iostream>
+using namespace std;
+
+
+// Create a ship class with some attributes
+class Ship {
+  public:
+	string nameofShip;
+	char orientation;
+	int xCoor;
+	int yCoor;
+};
+
+int main() {
+  // Create a Ship 1
+  Ship shipPlacement1;
+  shipPlacement1.nameofShip="Conquerer";
+  shipPlacement1.orientation;
```

Comment:
What did you intend to do on this line?  You need to create a function to set it